pandas/src/period.pyx

```diff
@@ -568,7 +565,7 @@ cdef _reso_local(ndarray[int64_t] stamps, object tz):
                                               &dts)
             dt = datetime(dts.year, dts.month, dts.day, dts.hour,
                           dts.min, dts.sec, dts.us, tz)
-            delta = int(total_seconds(_get_utcoffset(tz, dt))) * 1000000000
+            delta = int(timedelta.total_seconds(_get_utcoffset(tz, dt))) * 1000000000
```

Comment:
yes if it can completely be replaced (which i think it can) ; always nice to remove code